.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,67 +1,44 @@
 ---
-name: Bug report
+name: Library-specific Bug Report
 about: Report a problem with this library that needs to be fixed.
 title: ''
 labels: bug, triage
 assignees: ''
 
 ---
-<!--
-**PLEASE READ**
-==============================================================================
-Before posting, make sure you've carefully reviewed the documentation for this
-library:
-
-https://developers.google.com/google-ads/api/docs/client-libs/python
-
-This library is only used for the Google Ads API. For the AdWords API client
```

Comment:
I know you are trying to harmonize the text, but I would prefer to leave this in as we have regularly received questions here about the wrong API.

Reply:
Hm ok. There was some back and forth about this in the issue template review where other client libs have not seen this much, or think it sounds redundant with the rest of the "library-specific phrasing" but I think we can keep it here and I'll label it as optional in the main template.

Reply:
Actually, since we've added the templates with this "Google Ads API"-specific instruction, let's wait and see if this is needed. Is that fine with you?

Reply:
Sure, but if there's another AdWords API question here, I'm going to be insufferable.

Reply:
I will expect nothing less.